scripts/archive_api.py

```diff
@@ -512,14 +708,87 @@ async def stream_message_list(archiveClient, db_records, next_offset):
+                                                    "the offset in a following request to fetch the "
+                                                    "next block of messages."
+                                 },
+                                 "messages": {
+                                     "type": "array",
+                                     "items": messageRecordSchema,
+                                 }
+                             }
+                         }
+                     }
+                 }
+             },
+             400: {
+                 "description": "Bad request. This may be caused by an an invalid time range or "
+                                "requesting a non-existent topic.",
+                 "content": {
+                     "text/plain": {
+                         "schema": {
+                             "type": "string",
+                             "examples": ["Invalid time range", 
+                                          "Invalid message count limit", 
+                                          "Invalid message offset"],
+                         }
+                     }
+                 }
+             },
+             401: {
+                 "description": "Authentication required.",
+                 "content": {
+                     "text/plain": {
+                         "schema": {
+                             "type": "string",
+                             "examples": ["Authentication required"],
+                         }
+                     }
+                 }
+             },
+             403: {
+                 "description": "Not authorized. The authenticated user does not have access to the "
+                                "requested topic.",
+                 "content": {
+                     "text/plain": {
+                         "schema": {
+                             "type": "string",
+                             "examples": ["Operation not permitted"],
+                         }
+                     }
+                 }
+             },
+         })
 async def fetch_time_range(topic_name: Annotated[str, 
-                           Path(title="The name of the topic from which to read")],
-                           start: Annotated[int, Query(title="Timestamp for start of time range")],
-                           end: Annotated[int, Query(title="Timestamp for end of time range")],
-                           limit: Annotated[int, Query(title="Maximum number of messages to return at a tim")] = 10,
-                           offset: Annotated[int, Query(title="Offset of first message to return")] = 0,
-                           authorization: Annotated[Union[str, None], Header()] = None,
+                           Path(description="The name of the topic from which to read")],
+                           start: Annotated[int, Query(description="Timestamp for start of time range")],
```

Comment:
Or are these parameters auto-included in the docs? If so disregard my comment above.